hack/generated/pkg/genruntime/property_bag_test.go

```diff
@@ -6,23 +6,113 @@
 package genruntime
 
 import (
-	. "github.com/onsi/gomega"
-
 	"testing"
+
+	. "github.com/onsi/gomega"
 )
 
+/*
+ * NewPropertyBag() Tests
+ */
+
+func TestPropertyBag_NewPropertyBag_WithNoParameters_ReturnsEmptyBag(t *testing.T) {
+	g := NewWithT(t)
+	bag := NewPropertyBag()
+	g.Expect(bag).To(BeEmpty())
+}
+
+func TestPropertyBag_NewPropertyBag_WithZeroBag_ReturnsEmptyBag(t *testing.T) {
+	g := NewWithT(t)
+	var zeroBag PropertyBag
+	bag := NewPropertyBag(zeroBag)
+	g.Expect(bag).To(BeEmpty())
+}
+
+func TestPropertyBag_NewPropertyBag_WithEmptyBag_ReturnsEmptyBag(t *testing.T) {
+	g := NewWithT(t)
+	emptyBag := NewPropertyBag()
+	bag := NewPropertyBag(emptyBag)
+	g.Expect(bag).To(BeEmpty())
+}
+
+func TestPropertyBag_NewPropertyBag_WithSingleItemBag_ReturnsBagWithExpectedLength(t *testing.T) {
+	g := NewWithT(t)
+	original := NewPropertyBag()
+	g.Expect(original.Add("Answer", 42)).To(Succeed())
+
+	bag := NewPropertyBag(original)
+	g.Expect(bag).To(HaveLen(1))
+}
+
+func TestPropertyBag_NewPropertyBag_WithMultipleItemBag_ReturnsBagWithExpectedContent(t *testing.T) {
+	g := NewWithT(t)
+	original := NewPropertyBag()
+	g.Expect(original.Add("Answer", 42)).To(Succeed())
+	g.Expect(original.Add("Halloween", "31OCT")).To(Succeed())
+	g.Expect(original.Add("Christmas", "25DEC")).To(Succeed())
+
+	bag := NewPropertyBag(original)
+	g.Expect(bag).To(HaveKey("Answer"))
```

Comment:
You aren't asserting that the actual content is correct here, just that it has the key?
Shouldn't you also check to make sure that the key maps to the correct value?

Same comment applies below in a couple other tests, but won't repeat.

Reply:
Hmm I see in the other tests (that you already had) that you may have those cases covered already.

Maybe this just becomes minor: rather than `ReturnsBagWithExpectedContent` use `ReturnsBagWithExpectedKeys` to make it clear you only care about asserting on keys here?

Reply:
Other tests check that the values stored can be recalled cleanly, so I'm only worried in this test whether the expected keys are put into the bag. I'll rename the tests as you suggest.